executor/partition_table_test.go

```diff
@@ -222,6 +223,46 @@ func (s *partitionTableSuite) TestPartitionInfoDisable(c *C) {
 	tk.MustQuery("select * from t_info_null where (date = '2020-10-02' or date = '2020-10-06') and app = 'xxx' and media = '19003006'").Check(testkit.Rows())
 }
 
+func (s *partitionTableSuite) TestNewCollation(c *C) {
```

Comment:
It's not serial.

Reply:
You are right. Now I attach it to `testIntegrationSerialSuite` like other cases about collation.